microsoft-edge/privacy-whitepaper/index.md

```diff
@@ -103,6 +103,15 @@ You can delete templates stored on the device and clear the cache data. Go to `e
 You can use the title of your collection to find relevant Pinterest Topic pages when searching in Microsoft Bing. Go to `edge://settings/privacy`. Turn on the **Show suggestions from Pinterest in Collections** setting. Microsoft Edge does not send data about your collections to Pinterest. You can remove the suggestions and stop searches for Pinterest Topic pages. Go to `edge://settings/privacy` and turn off the **Show suggestions from Pinterest in Collections** setting.
 Collections aren't available when using **InPrivate** or **Guest** modes.
 
+<!-- ====================================================================== -->
+## Compose
+
+In the Bing side pane, Compose uses AI to write for you starting from a short prompt and "tone", "format", and "length" selections. Text generated by Compose can be copied or inserted into the text page directly. To access Compose, click on the blue Bing icon in the right hand pane > Compose (requires Bing waitlist approval). Text and selections inputed by the user is sent to an internal Microsoft Large Language Model (LLM).
+
+![Compose](./media/compose.png)
+
+The request is sent over a secure HTTPS connection with encryption and contains the input prompt, formatting selections, along with IDs associated with the user's Edge profile and device, to reference for subsequent requests. User input is not stored at all. 
```

Comment:
```suggestion
The request is sent over a secure HTTPS connection with encryption. The request contains the input prompt, formatting selections, and IDs that are associated with the user's Microsoft Edge profile and device, to reference for subsequent requests. User input is not stored at all. 
```
Opportunity to format as short sentences for increased readability, b/c two distinct thoughts.

https://learn.microsoft.com/en-us/style-guide/a-z-word-list-term-collections/t/that-vs-which -- "Include _that_ even if the sentence is clear without it. It helps to clarify the sentence structure."

Reply:
@majenny - per dandclark, you could omit ", which uses encryption." 
rendered context: 
https://github.com/MicrosoftDocs/edge-developer/blob/user/majenny/ComposePrivacy/microsoft-edge/privacy-whitepaper/index.md#compose